polaris/polaris/templates/polaris/deposit.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %} {% load i18n %}
+{% extends "./base.html" %} {% load i18n %}
```

Comment:
Should be extending from `polaris/base.html`. Otherwise the Anchor can't override `polaris/base.html`. I'm not sure about that though.

Reply:
Thats not the case, Django's `extends` keyword resolves relative paths and is able to detect when it is overridden by another app regardless of the hard-coded path. Django compares all paths relative to their respective app's `templates` folder: https://docs.djangoproject.com/en/3.1/ref/templates/api/#django.template.loaders.app_directories.Loader